packages/hs-react-ui/src/components/TextInput/TextInput.tsx

```diff
@@ -144,14 +145,14 @@ const defaultCallback = () => {}; // eslint-disable-line @typescript-eslint/no-e
 
 const TextInput = ({
   // Destructure native HTML attributes to provide default values
-  defaultValue = '',
   type = 'text',
   disabled = false,
   cols = 10,
   rows = 10,
 
   debouncedOnChange = defaultCallback,
   onClear,
+  onChange,
```

Comment:
what's the advantage of pulling this out of the nativeHTMLAttributes?

Reply:
I had this within before and things went awry. When it was NOT destructed out, customizing the onChange on InputComponent would always get overwritten when spreading the native props at the _end_ of the listed props. We could keep it in the nativeAttributes, ie not destruct it out, and make sure we add our customized props for the Input Component, including onChange, after the spread of the props so the custom props take precedence. I will remove the destructed onChange, and instead move the {...nativeAtrributes} to the first listed prop on InputComponent

Reply:
I think your way works just fine